### ISSUE TITLE
Issues following the docs for deploying feast on GCP using Terraform

### ISSUE THREAD

**Issue by josegpg** [feast-dev/feast#1419]
Hello!

I'm trying to test feast but I've found few issues when trying to deploy the platform on GCP through Terraform following the [docs](https://docs.feast.dev/getting-started/install-feast/google-cloud-gke-with-terraform). I'll list them below.

1. Helm fails to find the feast chart (since it was changed to a different [repository](https://github.com/feast-dev/feast-helm-charts). You can see [here](https://github.com/feast-dev/feast/blob/master/infra/terraform/gcp/feast.tf#L116) a reference to the old path.
2. Once I manually changed variable to point to the right location of the chart, however on the last terraform step (helm release) it's giving me timeout. When doing a small research I found that there's a Pod (feast-serving) entering a `CrashLoopBackOff` state, when looking at the logs I found the following:

> `Factory method 'servingServiceV2' threw exception; nested exception is io.lettuce.core.RedisConnectionException: Unable to connect to feast-0-8-redis-master:6379`

Doing some more research I found that the host should not be `feast-0-8-redis-master` since it should be overwritten [here](https://github.com/feast-dev/feast/blob/master/infra/terraform/gcp/feast.tf#L43) and that's not happening since the name of the variables to overwrite (found [here](https://github.com/feast-dev/feast/blob/master/infra/terraform/gcp/feast.tf#L39)) is `feast-online-serving` instead of `feast-serving` (see [here](https://github.com/feast-dev/feast-helm-charts/tree/main/charts/feast-serving)).

## Expected Behavior 

The terraform apply works completely.

## Current Behavior

Failing on the last step (helm release)

## Steps to reproduce

Basically follow [this](https://docs.feast.dev/getting-started/install-feast/google-cloud-gke-with-terraform) documentation from scratch.

### Specifications

- Version: 0.9.3
- Platform: GKE

## Possible Solution

I think the solution comes in 3 steps:
1. Update docs to reflect the new feast helm repo structure
2. Update terraform file to point to the new feast helm repo
3. Update overwrite vars with the proper chart name (`feast-serving`) so the redis ip is properly set and `feast-serving` pod can connect to redis.

I've tested (2 and 3) locally and finally got feast working on my GCP account. I'll create a sample fix PR since it's not too complicated

**Comment by woop** [comment]
Hey @josegpg. Thanks for the update and thanks for figuring out the problem! I've left a comment on your PR
